excess_reported_income_sources.md

```diff
@@ -0,0 +1,84 @@
+# Excess Reported Income specific formats and fund information
+
+<details>
+    <summary>🏦 Instructions for ERI_RAW format</summary>
+
+You will need:
+
+-   **CSV using the ERI_RAW format.** This is currently the only format supported for excess reported income.
+    [See example](https://github.com/KapJI/capital-gains-calculator/blob/main/cgt_calc/resources/eri/vanguard_eri.csv)
+
+Example usage for the tax year 2024/25:
+
+```shell
+cgt-calc --year 2024 --eri-raw-file eri_raw.csv [broker_transactions_options...]
```

Comment:
Add a comment that for Vanguard that file is loaded automatically?

Reply:
Yeah we should add at the top of the file that this guide is needed when the fund is not already bundled. I switched between several providers so I will be adding later more coverage. We will have not just Vanguard.